app/workers/zomato_data_grabbing_worker.rb

```diff
@@ -0,0 +1,73 @@
+require 'json'
+class ZomatoDataGrabbingWorker
+  include Sidekiq::Worker
+  include Faraday
```

Comment:
What does including Faraday allows you? (Asking out of curiosity, never saw this pattern)

Reply:
Faraday is used as networking\api addressing infrastructure
it is part of the exrercise guide